phpunit tests are failing in nighlty

The following tests are failing in [nightly](https://drone.owncloud.com/owncloud/openidconnect/1915/5/5)

```bash
There was 1 error:

1) OCA\OpenIdConnect\Tests\Unit\EventHandlerTest::testListenerDifferentEvent
PHPUnit\Framework\MockObject\UnknownTypeException: Class or interface "Symfony\Component\EventDispatcher\Event" does not exist

/var/www/owncloud/server/apps/openidconnect/tests/unit/EventHandlerTest.php:67
phpvfscomposer:///var/www/owncloud/server/lib/composer/phpunit/phpunit/phpunit:106

ERRORS!
Tests: 113, Assertions: 191, Errors: 1.

Generating code coverage report in Clover XML format ... done [00:00.008]
make: *** [Makefile:108: test-php-unit-dbg] Error 2
```

Related core issue: https://github.com/owncloud/core/issues/39630

--- a/tests/unit/EventHandlerTest.php
+++ b/tests/unit/EventHandlerTest.php
@@ -30,7 +30,7 @@ use OCP\SabrePluginEvent;
 use PHPUnit\Framework\MockObject\MockObject;
 use Sabre\DAV\Auth\Plugin;
 use Sabre\DAV\Server;
-use Symfony\Component\EventDispatcher\Event;
+use Symfony\Contracts\EventDispatcher\Event;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Test\TestCase;
 

